api/apiclient_test.go

```diff
@@ -60,19 +61,19 @@ func (s *apiclientSuite) TestConnectWebsocketMultiple(c *gc.C) {
 
 	// Check that we can use the proxy to connect.
 	info.Addrs = []string{proxy.Addr()}
-	conn, _, err := api.ConnectWebsocket(info, api.DialOpts{})
+	conn, _, err := api.DialAPI(info, api.DialOpts{})
 	c.Assert(err, jc.ErrorIsNil)
 	conn.Close()
-	assertConnAddrForEnv(c, conn, proxy.Addr(), s.State.ModelUUID(), "/api")
+	assertConnAddrForModel(c, conn, proxy.Addr(), s.State.ModelUUID())
```

Comment:
Also going to claim "broken windows principle" here. I'm still not sure if it's a precondition or not, but either way, it would be a small change to remove that line or split the test into two.